### PR TITLE
feat: Add PongWars sample

### PR DESCRIPTION
**Pull request by morning4coffe-dev**
Add a sample for UnoPongWars minigame.

### CODE REVIEW

File: UI/PongWars/UnoPongWars/UnoPongWars/Styles/ColorPaletteOverride.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
+//  This file is generated by a tool from the file ColorPaletteOverride.zip -- YOU SHOULD NOT EDIT IT manually.
```
Comment (morning4coffe-dev) [inline UI/PongWars/UnoPongWars/UnoPongWars/Styles/ColorPaletteOverride.cs:1]:
```suggestion
```
We can remove this line, fixed by https://github.com/unoplatform/uno.templates/pull/672

Comment (agneszitte) [reply UI/PongWars/UnoPongWars/UnoPongWars/Styles/ColorPaletteOverride.cs]: Done

**Comment by agneszitte** [comment]
@jeromelaban, @dansiegel right now I can see this [`error : The platform 'AnyCPU' is not supported for Self Contained mode.`](https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=124342&view=logs&j=e4c53f19-1dc0-5952-d087-a4a24555bd84&t=98a1b098-9f98-5127-03a6-7ce8a398f30f&l=66) for the CI Build and locally for release as well.

Does something need to be fixed in Templates? 
@morning4coffe-dev created a new solution on Monday (related commit https://github.com/unoplatform/Uno.Samples/pull/675/commits/f17249ceb83461340b6870a28e1590dbe9280627).
Not sure if it was like that by default. @morning4coffe-dev did you change some things manually?

**Comment by morning4coffe-dev** [comment]
agneszitte, I didn't change anything in the configuration when creating a new project from the template via wizard. I can reproduce the issue when creating a new Recommended project with 'VS Extension 5.2.46.54'

**Comment by MartinZikmund** [comment]
@agneszitte @morning4coffe-dev That error is fixed in the single project branch - I suggest just cherry-picking this commit, as this should be merged sooner - https://github.com/unoplatform/Uno.Samples/commit/81817c257bae7ac3fea7cb945bbdd53973f7795a

**Comment by MartinZikmund** [comment]
Rebased on latest master which includes the fix, this should now pass CI

**Review by MartinZikmund — APPROVED** [review]

**Comment by agneszitte** [comment]
> Rebased on latest master which includes the fix, this should now pass CI

Great for the CI Build, but locally for release what do we do @MartinZikmund, @jeromelaban?

**Review by nickrandolph — APPROVED** [review]

**Pull request merged**